README.md

```diff
@@ -1,32 +1,62 @@
-# Project name
+# Nucmorph-Colorizer 🔬🎨
 
-One line description
-
----
+#### A web-based, time-series visualizer for tracked segmented data.
 
 ## Description
 
-Main features in a brief descriptive text.
+**Nucmorph-Colorizer is a web tool for interacting with and visualizing features in time-series segmented data**. You can apply color maps and ranges,
+switch between features in your dataset, play through your data to observe motion, and view plots showing how feature data change over time!
+
+This project originated from the [Allen Institute for Cell Science (AICS)](https://alleninstitute.org/division/cell-science/) Nuclear Morphogenesis
+project and is being updated to support broader use cases. View our [Issues page](https://github.com/allen-cell-animated/nucmorph-colorizer/issues)
+for more details about potential future features!
+
+**[A hosted version of the app can be accessed here!](http://dev-aics-dtp-001.corp.alleninstitute.org/nucmorph-colorizer/dist/index.html)**
+
+![image](https://github.com/allen-cell-animated/nucmorph-colorizer/assets/30200665/d9d22cba-faa0-4366-a647-973bc2fce360)
+
 
 ## Installation
+Installing is optional! You can use the [hosted version of Nucmorph Colorizer](http://dev-aics-dtp-001.corp.alleninstitute.org/nucmorph-colorizer/dist/index.html)
+to access our existing datasets without installing the project. 
 
-Describe how to obtain the software and get it ready to run
+**Prerequisites:**
+- Node
```

Comment:
do you have a lowest node version you support? To be safe you could include whatever version you're currently building it with